BackofficeBundle/Twig/LeftPanelExtension.php

```diff
@@ -2,24 +2,17 @@
 
 namespace OpenOrchestra\BackofficeBundle\Twig;
 
-use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Class LeftPanelExtension
  */
-class LeftPanelExtension extends \Twig_Extension
+class LeftPanelExtension extends \Twig_Extension implements ContainerAwareInterface
```

Comment:
Can't you instead inject the open_orchestra_backoffice.left_panel_manager service ?